torch/_inductor/fx_passes/efficient_conv_bn_eval.py

```diff
@@ -70,6 +75,165 @@ def efficient_conv_bn_eval(
     return output
 
 
+def efficient_conv_bn_fn_eval(
```

Comment:
I can separate this pass in a different PR if that's easier to review. I would need to write a test to use ops like `torch.ops.aten.conv1d.default` directly instead of nn modules through.